Library/Formula/blazeblogger.rb

```diff
@@ -1,13 +1,18 @@
-require 'formula'
-
 class Blazeblogger < Formula
-  homepage 'http://blaze.blackened.cz/'
-  url 'https://blazeblogger.googlecode.com/files/blazeblogger-1.2.0.tar.gz'
-  sha1 '280894fca6594d0c0df925aa0a16b9116ee19f17'
+  homepage "http://blaze.blackened.cz/"
+  url "https://blazeblogger.googlecode.com/files/blazeblogger-1.2.0.tar.gz"
+  sha1 "280894fca6594d0c0df925aa0a16b9116ee19f17"
 
   def install
-    system "make", "prefix=#{prefix}",
-                   "compdir=#{prefix}",
-                   "install"
+    ENV.deparallelize # Build fails on Mavericks
```

Comment:
This is probably not specific to Mavericks; can you remove the comment and report this upstream if you can repro it? Though it doesn't look like this has seen a commit since 2011... https://code.google.com/p/blazeblogger/issues/list